src/main/java/slimeknights/tconstruct/tools/data/MaterialIds.java

```diff
@@ -5,61 +5,54 @@
 
 public final class MaterialIds {
 
-  // natural resources/blocks
+  //Tier 1
   public static final MaterialId wood = id("wood");
-  public static final MaterialId stone = id("stone");
-  public static final MaterialId flint = id("flint");
-  public static final MaterialId cactus = id("cactus");
   public static final MaterialId bone = id("bone");
-  public static final MaterialId obsidian = id("obsidian");
-  public static final MaterialId prismarine = id("prismarine");
-  public static final MaterialId endstone = id("endstone");
+  public static final MaterialId stone = id("stone");
   public static final MaterialId paper = id("paper");
```

Comment:
I have some thoughts on paper I want to talk about on Discord when you are next online